apps/site/assets/ts/stop/components/Pill.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { ReactElement } from "react";
+
+export const bgClass = (color: string | undefined): string =>
+  color ? `u-bg--${color}` : "";
+
+const Pill = ({
+  onClick,
+  selected = false,
+  backgroundColor,
+  optionalCSS = "",
+  children
+}: {
+  onClick?: () => void;
+  selected?: boolean;
+  backgroundColor?: string | undefined;
+  optionalCSS?: string | undefined;
+  children: ReactElement<HTMLElement> | string;
+}): ReactElement<HTMLElement> => {
+  return (
+    <div
```

Comment:
Getting an ESLint error on this:

> Avoid non-native interactive elements. If using native HTML is not possible, add an appropriate role and support for tabbing, mouse, keyboard, and touch inputs to an interactive content element  [jsx-a11y/no-static-element-interactions](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/main/docs/rules/no-static-element-interactions.md)

**suggestion:** update this tag to a `<button>`.

Reply:
The other ESLint error I _think_ will also resolve itself if you change the tag to a button:

> Visible, non-interactive elements with click handlers must have at least one keyboard listener                                                                                                     [jsx-a11y/click-events-have-key-events](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/main/docs/rules/click-events-have-key-events.md)